src/preprocessing/passes/int_to_bv.cpp

```diff
@@ -226,7 +226,6 @@ Node intToBV(TNode n, NodeMap& cache)
           {
             Rational constant = current.getConst<Rational>();
             if (constant.isIntegral()) {
-              AlwaysAssert(constant >= 0);
```

Comment:
Don't other parts of this pp assume all constants are non-negative? For example, is the case of translating `<` and `<=` robust enough for negative constants?

Reply:
Yes, it should be the same as `UMINUS` of a positive constant (without the widening by one bit because it is not necessary).